src/docs/product/accounts/membership.mdx

```diff
@@ -60,36 +61,69 @@ Member roles dictate access within an organization.
+| Can invite new members to join (_Invite will land in pending state for managers and owners to approve_) | X                                                  | X                    |
+| Can add and remove teams                                                                                |                                                    |                      |
+| Can add and remove repositories                                                                         | X (_Only for teams they’re an existing Admin for_) |                      |
+| Can change organization settings                                                                        |                                                    |                      |
+| Can remove an organization                                                                              |                                                    |                      |
+| Can assign Team Admin Role                                                                              | X (_Only for teams they’re an existing Admin for_) |                      |
+| Can add, edit, or remove Team Contributors                                                              | X                                                  |                      |
+| Can add Projects for teams (depending on the rules set by the org owner or manager)                     | X                                                  | X                    |
+| Can remove Projects from teams                                                                          | X                                                  |                      |
+
+### Team Admin
+
+- Admin access to teams of which they're members.
+- Can create new teams and Projects, as well as remove Projects and teams of which they're members.
+
+### Team Contributor
+
+- Can view and act on issues.
+- Can join and leave teams (depending on the rules set by the org owner/manager).
+- Can invite new members to join (pending approval from org owner/manager).
+- Can add Projects for team (depending on the rules set by the org owner/manager).
+
 ## Open Membership
 
-Open membership is enabled by default, which allows members to join, leave, and add other members to teams.
+Open membership is enabled by default. This allows users to join, leave, and add other members to teams.
 
 ## Restricting Access
 
-- Access to a Project is limited to the Team that owns the project. However, with open membership,
-  members can join and add members to teams.
-- To further restrict access to a Project, you can control access to Teams by making them accessible
-  only through invitation by a Manager, Owner, or Admin.
+- Project access is limited to the team that owns the Project, but org members can get access to the Project by joining the team that owns it (depending on the rules set by the org owner/manager).
```

Comment:
> depending on the rules set by the org owner/manager

This needs to be specific. If I were an org-owner and I want to stop people from joining teams, then I'll need to go figure out which Sentry rules need to be changed. 

Can we say "depends on Open Membership setting"?

Reply:
There's overlap with the explanation for Team Access below.

I think we need to explain the relationship between Team vs Project, and tie that together in the explanation for the edge-case above.